ALUM/ALUM/Services/APIConfig.swift

```diff
@@ -0,0 +1,104 @@
+//
```

Comment:
This file will host IN ONE PLACE all details we need about the backend routes including the BASE URL + routes + error codes for the routes etc.

Every time a backend change is introduced that affects frontend this is the file that will need updates for sure.

@adhi0331 @christen03 fyi